src/index.ts

```diff
@@ -0,0 +1,121 @@
+import { type RemotePinningServiceClient, type Pin, type PinStatus, type PinsRequestidPostRequest, Status } from '@ipfs-shipyard/pinning-service-client'
+import { multiaddr } from '@multiformats/multiaddr'
+import debug from 'debug'
+import pRetry from 'p-retry'
+import { FailedToConnectToDelegates } from './errors.js'
+import type { Helia } from '@helia/interface'
+import type { CID } from 'multiformats/cid'
+
+const log = debug('helia-remote-pinning')
+const logError = log.extend('error')
+const logTrace = log.extend('trace')
+
+export interface HeliaRemotePinningOptions {
```

Comment:
I don't think this needs to be exported, it's only used in the factory method sig and it's not used in the method body.